core/main/src/broker/endpoint_broker.rs

```diff
@@ -404,6 +414,43 @@ impl EndpointBrokerState {
         }
     }
 
+    fn handle_static_ruquest(
+        &self,
+        rpc_request: RpcRequest,
+        extn_message: Option<ExtnMessage>,
+        rule: Rule,
+        callback: BrokerCallback,
+    ) {
+        let mut data = JsonRpcApiResponse::default();
```

Comment:
can instansiation of `data` be moved inside of `let` to avoid creating when the let does not match?

Reply:
done